### PR TITLE
ENH: Add support for conda-installed FFmpeg on macOS

### PR DESCRIPTION
**Pull request by hoechenberger**
On macOS, pyglet wasn't able to detect the FFmpeg libraries installed into a `conda` environment, as `conda` doesn't add them to `LD_LIBRARY_PATH`. Instead, one has to consider the `CONDA_PREFIX` environment variable, which points to the currectly active `conda` environment.

### CODE REVIEW

File: pyglet/lib.py [hunk 1]
```diff
@@ -222,6 +222,11 @@ def find_library(self, path):
                 sys.frozen is True and pyglet.compat_platform == 'darwin'):
             search_path.append(os.path.join(sys._MEIPASS, libname))
 
+        # conda support
+        if os.environ.get('CONDA_PREFIX', False):
+            search_path.append(os.path.join(os.environ['CONDA_PREFIX'], 'lib',
```
Comment (benmoran56) [inline pyglet/lib.py]: If you prefer, we allow a line length of 100 columns in pyglet.

Comment (hoechenberger) [reply pyglet/lib.py]: Thanks, I've changed this in e0ca014

**Comment by benmoran56** [comment]
I thought I might try to change the base from `master` to `pyglet-1.4-maintenance` on my end, but that doesn't work so well. No worries, I'll merge this in manually later today. 

Thanks again!

**Comment by hoechenberger** [comment]
Thank you very much, @benmoran56!

**Comment by benmoran56** [comment]
Merged into master for v2.0, as well as 1.4-maintance branch for the upcoming v1.4.7 point release. Thanks again!

**Pull request merged**